README.md

```diff
@@ -1,4 +1,22 @@
-## Inventory
+# Inventory
+
+Demo app for showing how Ditto sync and counters work.
```

Comment:
I would say "Demo app for showcasing Ditto's real-time sync and Conflict Resolution through the use of an inventory counter."